patches/minecraft/net/minecraft/network/NetHandlerPlayServer.java.patch

```diff
@@ -34,6 +34,24 @@
                  this.field_147367_d.func_184103_al().func_148544_a(itextcomponent, false);
              }
  
+@@ -1049,14 +1056,14 @@
+         if (entity != null)
+         {
+             boolean flag = this.field_147369_b.func_70685_l(entity);
+-            double d0 = 36.0D;
++            float d0 = net.minecraftforge.common.ForgeHooks.getPlayerReachDistance(this.field_147369_b) + this.field_147369_b.func_70047_e() + (float)Math.max(entity.func_174813_aQ().field_72336_d - entity.func_174813_aQ().field_72340_a, Math.max(entity.func_174813_aQ().field_72337_e - entity.func_174813_aQ().field_72338_b, entity.func_174813_aQ().field_72334_f - entity.func_174813_aQ().field_72339_c));
```

Comment:
Maybe it'd be better to square this than forcing a sqrt to be taken later on? Also would reduce patch size.

Reply:
Yeah, not a bad idea. Doing that now.